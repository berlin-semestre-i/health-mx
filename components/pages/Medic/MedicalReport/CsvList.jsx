import React from 'react'

const List = ({array}) => (
  array.length > 0?
    <React.Fragment>
      {array.map((item, index) => (
        <span key={index}>
          {item}{index < array.length-1 && ', '}
        </span>
      ))}
    </React.Fragment> :
    ('Ninguno')
)

export default List