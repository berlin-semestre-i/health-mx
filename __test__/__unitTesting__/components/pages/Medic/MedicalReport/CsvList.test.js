import React from 'react'
import { shallow } from 'enzyme'
import CsvList from '../../../../../../components/pages/Medic/MedicalReport/CsvList'

describe('CsvList', () => {
  it('renders', () => {
    const component = shallow(<CsvList array={[]} />)
    expect(component).toMatchSnapshot()
  })
})