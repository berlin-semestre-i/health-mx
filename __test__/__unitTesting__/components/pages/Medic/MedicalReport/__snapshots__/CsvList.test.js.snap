// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CsvList renders 1`] = `"Ninguno"`;
